### PR TITLE
[sailfish-browser] Filter out accidental clicks near the display edge. Fixes JB#47839

### PR DESCRIPTION
**Pull request by jpetrell**
…. Fixes JB#47839

### CODE REVIEW

**Comment by jpetrell** [comment]
Normally we use Theme.paddingLarge, but since the main browser controls are located right at the edge don't want to reduce the reactive area too much.

File: src/pages/components/IconButton.qml [hunk 1]
```diff
@@ -32,5 +33,9 @@ IconButton {
         }
     }
 
+    onPressed: DragFilter.begin(mouse.x, mouse.y)
+    onCanceled: DragFilter.end()
+    DragFilter.screenMargin: Theme.paddingMedium
```
Comment (rainemak) [inline src/pages/components/IconButton.qml]: Not related to this but noticed while testing that tab icon and menu icon do not highlight when tapping near the bottom corner.

File: src/pages/components/ToolBar.qml [hunk 2]
```diff
@@ -264,6 +265,10 @@ Column {
                 }
             }
 
+            DragFilter.screenMargin: Theme.paddingMedium
+            onPressed: DragFilter.begin(mouse.x, mouse.y)
+            onCanceled: DragFilter.end()
```
Comment (rainemak) [inline src/pages/components/ToolBar.qml]: Nit (of nits): Different order than IconButton :)

**Comment by rainemak** [comment]
Noticed a highlight issue but that's not related to this one PR se. LGTM :+1:

File: src/pages/components/ToolBar.qml (hunk 2 above)
Comment (jpetrell) [reply src/pages/components/ToolBar.qml]: Fixed.

**Pull request merged**